src/config/preferences.c

```diff
@@ -276,11 +276,7 @@ prefs_reload(void)
 void
 prefs_load(const char* config_file)
 {
-    if (config_file == NULL) {
-        load_config_keyfile(&prefs_prof_keyfile, FILE_PROFRC);
-    } else {
-        load_custom_keyfile(&prefs_prof_keyfile, g_strdup(config_file));
-    }
+    load_custom_keyfile(&prefs_prof_keyfile, g_strdup(config_file ?: FILE_PROFRC));
```

Comment:
load_config_keyfile() gets the path xdg config path. load_custom_keyfile doesnt do that afaik